preprocess/analysis/language_analysis_tests.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jun 23 16:36:57 2020
+
+@author: robert
+"""
+import unittest
+from language_analysis import analyze_entities, analyze_sentiment
+import json
+
```

Comment:
Some discussion on the python style guide for ordering import statements: https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement
Looks like you don't **have** to alphabetize imports according to PEP, but it's kind of the standard

Reply:
<3

Reply:
Just use a python linter and you shouldn't have to worry about it